flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescalingITCase.java

```diff
@@ -670,12 +676,15 @@ public void run(SourceContext<Integer> ctx) throws Exception {
 				}
 
 				Thread.sleep(100);
```

Comment:
This seems like a long wait time actually (100ms). How about waiting 2ms and signaling after the 10th element.